crypto/tls/test/tls_integration_test.go

```diff
@@ -145,14 +145,14 @@ func newIntegrationClientServer(
 				if err == nil {
 					return false
 				}
-				return strings.Contains(err.Error(), "connection reset by peer") || strings.Contains(err.Error(), "broken pipe")
+				return strings.Contains(err.Error(), "connection reset by peer") ||
+					strings.Contains(err.Error(), "broken pipe") ||
+					strings.Contains(err.Error(), "client conn could not be established")
 			}
-			for i := 0; i < 3 && isRST(err) && tc.httpExpectError != nil; i++ {
+			for i := 0; i < 5 && isRST(err) && tc.httpExpectError != nil; i++ {
```

Comment:
Let's try a few more attempts, since this test code was flaking a ton under v1.24.